history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -253,8 +270,15 @@ intreat that you will tenderly and charitably consider of the premises,
 and find out some effectual expedient or way for our relief from prisons, spoil,
 and ruin.
 
+--
+
+[.offset]
 The following application was likewise presented.
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
```

Comment:
So this is a little tricky...a little further down on line 299 there's something that seems to be another [.letter-heading] but I'm not sure if these two things are the same document or not....they seemed to be to me in the original, but maybe I should have separated them.

Reply:
yup, tricky indeed. I stared it for a while feeling indecisive.  I think my best guess here is actually to treat this as _TWO_ embedded documents, because it sort of is, there's a first introductory document introducing the other document which they say is _annexed_. 

So, let's close out this embed at around line 298, and start another one.